docs/src/development/local/lando.md

```diff
@@ -4,43 +4,190 @@ sidebarTitle: "Lando"
+To have these variables set automatically, install the Platform.sh [CLI](../../gettingstarted/introduction/own-code/cli-install.md).
 
-If you do not yet have your project checked out, run:
+Alternatively, you can set the `$PLATFORMSH_CLI_TOKEN` and `$PLATFORMSH_SITE_NAME` environment variables manually.
 
-```bash
+Familiarity with the Lando [CLI commands](https://docs.lando.dev/cli/) is recommended but not necessary.
+
+Depending of your hardware, running the commands below can take a while.
+Check the [hardware requirements](https://docs.lando.dev/getting-started/installation.html#hardware-requirements) before attempting this guide.
+
+## Installation
+
+See the [Lando documentation](https://docs.lando.dev/getting-started/installation.html) for installing and setting up Lando on your system.
+
+### Get started with Lando
+
+The following examples are based on the [PHP template](https://github.com/platformsh-templates/php).
+
+Several stacks are available on Lando (LAMP, LEMP, MEAN).
+If you use a specific stack, adapt the steps below accordingly.
+See the [Lando documentation for all recipes and stacks available](https://docs.lando.dev/).
+
+Lando requires a `.lando.yml` file. The `init` command can generate that file automatically.
+
+The full output of the commands below have been redacted for brevity.
+
+**TODO: fix the Lando init step for both paths + remove the need for the prompt (just add a note) + add stack + add cloning of the PHP base project**
+
+{{< codetabs >}}
+
+---
+title=On a new project
+file=none
+highlight=false
+---
+
+Run `lando init --recipe platformsh --source platformsh` and follow the instructions provided by the interactive prompt.
+
+``` bash
 $ lando init --source platformsh
+
+? Select a Platform.sh account <email address of your account>
+? Which project? <project name>
 ```
 
-That will offer an interactive dialog to select the Platform.sh project to download, and then setup basic Lando configuration.
+<--->
+
+---
+title=On an existing project
+file=none
+highlight=false
+---
+
+Open a shell and access the directory where your project is located.
 
-If you already have your project checked out locally, change into the project directory and run:
+Enter `lando init --recipe platformsh --source cwd` and follow the instructions provided by the interactive prompt.
 
-```bash
-$ lando init --source cwd --recipe platformsh
+``` bash
+$ lando init --source platformsh --source cwd --webroot .
+? From where should we get your app's codebase? current working directory
+? What recipe do you want to use? platformsh
+? Select a Platform.sh account <email address of your account>
+? Which project? <project name>
 ```
 
-That will use your existing git clone and add the appropriate Lando configuration.
+{{< /codetabs >}}
 
-Either way, start the local environment by running:
+Once the command ran successfully, you get the following:
```

Comment:
As above, I'm not sure the specific output is as valuable as a description of what should happen.